src/Imgix/UrlBuilder.php

```diff
@@ -23,12 +23,27 @@ public function __construct($domains, $useHttps = false, $signKey = "", $shardSt
             throw new \InvalidArgumentException("UrlBuilder requires at least one domain");
         }
 
+        $this->validateDomain($this->domains);        
+
         $this->useHttps = $useHttps;
         $this->signKey = $signKey;
         $this->shardStrategy = $shardStrategy;
         $this->includeLibraryParam = $includeLibraryParam;
     }
 
+    private function validateDomain($domains) {
+        $DOMAIN_PATTERN = "/^(?:[a-z\d\-_]{1,62}\.){0,125}" .
+        "(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)[a-z\d]{1,63}$/";
```

Comment:
I think you should just put this all on one line, length-guidelines be damned. It's not like you're making it any more readable by splitting it between two.

Reply:
🙃